tests/runtests.bats

```diff
@@ -15,6 +16,8 @@ setup_file() {
 
     export VAGRANT_HOME=${TEMPDIR}/.vagrant.d
     echo "# Using ${VAGRANT_HOME} for VAGRANT_HOME" >&3
+
+    bash "${SCRIPT_DIR}/../tools/create_box_with_two_disks.sh" "${VAGRANT_HOME}" "${VAGRANT_CMD}"
```

Comment:
Do we need this? It's also getting called as part of the github workflow script in addition to being called below, the one below being just for the test that requires it which would be the preferred option until there is a published box that has two disks, though likely always useful to be able to prepare them in various forms.

Reply:
When we run in local the runtest.sh, we use a new temp directory and the box doesn't exist. 
I think the most solution, it is send box into box repository.

Reply:
I remove it.